### PR TITLE
New structure of docker images

### PR DESCRIPTION
**Pull request by petrbel**
Don't forget to set up dockerhub

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea/
```
Comment (FloopCZ) [inline .gitignore:1]: okeeey. :slightly_smiling_face:

File: dockerfiles/archlinux [hunk 2]
```diff
@@ -12,10 +12,10 @@ RUN pacman --noconfirm --needed -Syu \
       python \
       vim \
       python-numpy \
-      python-pandas \
-      python-scipy \
       python-pip \
-      sudo
+      sudo \
+      boost \
+      openssh
```
Comment (FloopCZ) [inline dockerfiles/archlinux:18]: We should decide whether we really always want `opencv` and `hdf5` (it was true so far).

**Review by FloopCZ — APPROVED** [review]
Thank you @petrbel. Now we need to migrate all our projects. :-)

File: dockerfiles/archlinux (hunk 2 above)
Comment (petrbel) [reply dockerfiles/archlinux:18]: I think we do

**Comment by petrbel** [comment]
Actually, since we pretty precisely tag every image, there's no reason to panic. All projects have fixed the versions and I'm not gonna delete a thing, so everything should work without any change. Only new builds with based on `latest` or tags newer than `2019-04-14` will have to adapt.

**Pull request merged**